charts/unleash-edge/README.md

```diff
@@ -0,0 +1,57 @@
+# Unleash Edge
+
+Unleash Edge offers a way to use Unleash in client-side applications, such as single page and native apps. It also offers a way to offload a main Unleash server, so that not every client refresh is a database operation.
+
+See the [reference documentation](https://github.com/unleash/unleash-edge/) for more on the reasons why you might want to deploy Edge in your cluster.
```

Comment:
Do we want to point to the docs or to github here? I'm fine with either (since they're the same content), but just want to make sure it's intentional.